src/index.html

```diff
@@ -80,38 +80,59 @@ <h4><a href="https://pub.dartlang.org/">Dart packages</a></h4>
         <div class="content">
           <h1>Why Dart?</h1>
           <p class="lead">
-            Dart has been used to ship many high-quality, mission-critical applications on the web, iOS, and Android at 
-            Google and elsewhere. It is a great fit for mobile and web development because it provides a number of key 
-            benefits highly relevant for client-side development.
+            Developers at Google and elsewhere use Dart to create
+            high-quality, mission-critical apps for iOS, Android, and the web.
+            With features aimed at client-side development,
+            Dart is a great fit for both mobile and web apps.
           </p>
         </div>
       </div>
       <div class="col-sm-7 col-lg-8">
         <div class="content">
+
           <h2 class="core-goals__first">Productive</h2>
-          <p>Syntax must be clear and concise, tooling simple, and dev cycles near-instant and on-device.</p>
-          <p>Dart increases developer velocity because it has a clear, succinct syntax and is able to run on a 
-            VM with a JIT compiler. The latter allows for <a href='https://flutter.io/hot-reload/'>stateful hot reload</a> 
-            during mobile development, resulting in super fast dev cycles, where you can edit code, compile and replace 
-            in the running app on the device.</p>
+          <p>
+            Dart’s syntax is clear and concise,
+            its tooling simple yet powerful.
+            Sound typing helps you to identify subtle errors early.
+            Dart has battle-hardened
+            <a href="/guides/libraries/library-tour">core libraries</a> and an
+            <a href="https://pub.dartlang.org/">ecosystem</a> of
+            thousands of packages.
+          </p>
+
           <h2>Fast</h2>
-          <p>Runtime performance and startup must be great and predictable even on small mobile devices.</p>
-          <p>With its ability to efficiently compile to native code ahead of time, Dart provides predictable, 
-            high performance and fast startup on mobile devices.</p>
+          <p>
+            Dart provides optimizing ahead-of-time compilation to get
+            predictably high performance and fast startup across
+            mobile devices and the web.
+          </p>
+
           <h2>Portable</h2>
-          <p>Client developers have to think about three platforms today: iOS, Android, and Web. The language needs to work 
-            well on all of them.</p>
-          <p>Dart supports compilation to native code (ARM, x86, etc.) for fast mobile performance as well as transpilation 
-            to efficient JavaScript for the web.</p>
+          <p>
+            Dart compiles to native ARM and x86 code, so that Dart mobile apps
+            can run natively on iOS, Android, and beyond.
```

Comment:
A little too much "native" in this sentence. I'd propose getting rid of the first one.

Reply:
Good catch.